crates/bevy_winit/src/winit_windows.rs

```diff
@@ -243,7 +246,11 @@ impl WinitWindows {
                 winit_window_attributes.with_min_inner_size(min_inner_size)
             };
 
-        #[allow(unused_mut)]
+        #[expect(clippy::allow_attributes, reason = "`unused_mut` is not always linted")]
+        #[allow(
+            unused_mut,
+            reason = "This variable needs to be mutable if `cfg(target_arch = "wasm32")`"
```

Comment:
I have a feeling that the quotation marks around "wasm32" are messing this string up. You might need to escape them?

Reply:
Woops...